.agents/skills/orchestrate-create-plan/SKILL.md

```
@@ -72,6 +98,9 @@ critiques drafts until the plan is complete.
    - flag any proposed scope substitution, such as replacing the user's named
      technology, API, SDK, product surface, or acceptance target with a different
      implementation surface;
+   - flag any proposed permission/tool-surface, workflow, public-contract, or
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 4; Actual: 3

(MD005, list-indent)